oauth_dropins/reddit.py

```diff
@@ -132,3 +131,9 @@ def get(self):
                       user_json=json_dumps(user_json))
     auth.put()
     self.finish(auth, state=st)
+
+def get_reddit_api(refresh_token):
```

Comment:
good idea! the standard method to override for this from `BaseAuth` is just named `api()`:

https://github.com/snarfed/oauth-dropins/blob/9aa714eb3292f72ecd8376e0955c3bf635e71c49/oauth_dropins/models.py#L46-L54

Reply:
yeah, i'd seen that but I wrote `get_reddit_api` as a static method outside of the class (sort of like `twitter_auth.py`).  is that OK? otherwise I have to load the RedditAuth object over on the granary side, which requires calling ndb datastore I think?

Reply:
ah, i see. the code reuse is pretty trivial, this helper isn't really adding much value, so i'd lean toward just instantiating `praw.Reddit` in both o-d and granary directly instead.

Reply:
OK, migrated this over to granary.